src/components/Contact.tsx

```diff
@@ -1,15 +1,37 @@
+import {useState} from 'react'
+import axios from 'axios';
+
 export default function Contact() {
+  const [name, setName] = useState('')
+  const [email, setEmail] = useState('')
+  const [message, setMessage] = useState('')
+
+  function onSubmit(event: any) {
+    event.preventDefault()
+
+    const formToBeSubmitted = {
+      name: name,
+      email: email,
+      message: message
+    }
+
+    axios.post("https://email-free-api.herokuapp.com/send-email", formToBeSubmitted)
+
+    //Cleaning input fields
+    setName(''); setEmail(''); setMessage('')
```

Comment:
Porque não deixar o formulário com os dados? Achei o comportamento de limpar muito abrupto, concorda?

Reply:
Well, its because I dont thought in a way to "confirm" the email sending. The cleaning of the input fields was the only way to confirm this, but with the "toast" hint that you wrote above, I can delete this cleaning input fields code.